src/sagemaker_xgboost_container/data_utils.py

```diff
@@ -528,33 +528,50 @@ def _get_pipe_mode_files_path(data_path: Union[List[str], str]) -> List[str]:
     return files_path
 
 
+def _make_symlinks_for_files_under_a_folder(dest_path: str, data_path: str):
+    if (not os.path.exists(dest_path)) or (not os.path.exists(data_path)):
+        raise exc.AlgorithmError("Unable to create symlinks as {data_path} or {dest_path} doesn't exist ")
+
+    logging.info("Making smlinks from folder {} to folder {}".format(data_path, dest_path))
```

Comment:
nit: Can use f-strings here instead of format for better readability.

Reply:
sure, Thanks